android/app/src/main/java/org/openbot/common/ControlsFragment.java

```diff
@@ -48,7 +48,6 @@ public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceStat
         .getWindow()
         .addFlags(android.view.WindowManager.LayoutParams.FLAG_KEEP_SCREEN_ON);
 
-    phoneController.init(requireContext());
```

Comment:
Why is this line removed? I believe it is causing the app to crash since the phoneController is not initialized.

Reply:
I wanted to eliminate the need to call `init()` because people forget to call it. Now we are forced to pass the context via the `getInstance(contect)` method.

This latest push should address the connectivity issues. The main problem was that I was returning a new connection from the `ConnectionManager` class when the user called `getConnection()`

Also updated the the latest version of the RTSP server.
In some rear cases, the video stream will fail to start. I will see if there is a fix, but in the mean time, just close the Free Roam fragment by going back, end enter it again.